tracks/adding-integration-tests-to-ansible-collections/track.yml

```diff
@@ -0,0 +1,283 @@
+      config file = None
+      configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
+      ansible python module location = /rusr/share/lib64/python3.6/site-packages/ansible
+      ansible collection location = /root/.ansible/collections:/usr/share/ansible/collections
+      executable location = /home/ansible/developing_colelction/bin/ansible
+      python version = 3.6.12 (default, Sep 15 2020, 12:49:50) [GCC 4.8.5 20150623 (Red Hat 4.8.5-37)]
+      jinja version = 2.11.3
+      libyaml = True
+    ~~~
+
+    You can verify ansible-core version from the first line of the above output.
+  tabs:
+  - title: Shell
+    type: terminal
+    hostname: shell
+  difficulty: basic
+  timelimit: 600
+- slug: clone-collection-that-requires-integration-tests
+  id: d3cye2zxcpyr
+  type: challenge
+  title: Choose a Collection that needs integration tests
+  teaser: '#'
+  assignment: |-
+    For this course, we'll use `community.postgresql` collection and write a basic integration test for the postgresql_info module.
+
+    Create the following directory for use of `community.postgresql` collection:
+
+    ```
+    mkdir -p ~/ansible_collections/community/postgresql
+    ```
+
+    Clone `community.postgresql` into the newly created directory:
+
+    ```
+    git clone https://github.com/ansible-collections/community.postgresql ~/ansible_collections/community/postgresql
+    ```
+
+    ```
+    cd ~/ansible_collections/community/postgresql
+    ```
+
+    Switch to `integration_example` branch:
+
+    ```
+    git checkout integration_example
+    ```
+
+    Verify that you're on `integration_example` branch by running the below command:
+
+    ```
+    git branch
+    ```
+  tabs:
+  - title: shell
+    type: terminal
+    hostname: shell
+  difficulty: basic
+  timelimit: 600
+- slug: cd-integration-directory
+  id: nsynktrioxvc
+  type: challenge
+  title: Create directory structure for integration tests
+  teaser: '#'
+  assignment: |-
+    You must place integration tests in the appropriate tests/integration/targets/ directory.
+
+    We will start with creating a setup target that will install all required packages and will run PostgreSQL service.
+
+    Change directory to Collection path:
+
+    ```
+    cd ~/ansible_collections/community/postgresql/
+    ```
+
+    Create the following directories:
+
+    ```
+    mkdir -p tests/integration/targets/setup_postgresql_db/tasks
+    ```
+
+    Create tests/integration/targets/setup_postgresql_db/tasks/main.yml and copy paste the below:
+
+    ```
+    vim tests/integration/targets/setup_postgresql_db/tasks/main.yml
+    ```
+
+    Note: Turn on `paste` mode in vim by typing `:set paste` and then paste the below text to avoid indendation errors.
+
+    ```yaml
+    - name: Install required packages
+      package:
+        name:
+          - apt-utils
+          - postgresql
+          - postgresql-common
+          - python3-psycopg2
```

Comment:
Note: This package set looks like it will only work on Debian-derivatives.  Is that a problem or should it be called out as a prerequisite for this track?